lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Migrations.hs

```diff
@@ -193,7 +220,7 @@ spec = describe "SHELLEY_MIGRATIONS" $ do
               testAddressCycling 3
               testAddressCycling 10
 
-    Hspec.it "SHELLEY_MIGRATE_01_big_wallet - \
```

Comment:
_hoping we can remove the retrying `it` soon_

Reply:
That's interesting! I had no idea that this difference even existed. Thanks for pointing it out.

I'm just thinking how we could make this difference clearer, so that the name of the function makes it obvious whether or not the contents will be retried on failure.

Reply:
Fixed in 7ada44899cf6028d4768c8c2403fa670fd00ec18.